.pre-commit-config.yaml

```diff
@@ -38,3 +38,7 @@ repos:
         language: system
         entry: ./check_readme.sh
         pass_filenames: false
+      - id: check-dependabot-coverage
+        name: Check if all actions are listed in dependabot.yml
+        language: system
+        entry: .github/actions/dependabot-coverage-check/dependabot-coverage-check.sh
```

Comment:
not sure it really make sense here given that we already have `generate-dependabot` before which will never make this to trigger.

Reply:
that's what I meant in another comment: the action is called, and if it fails, maybe one of those calls is buggy so it's nice to have this in a test.yml pipeline, no?